dolphinscheduler-task-plugin/dolphinscheduler-task-api/src/main/java/org/apache/dolphinscheduler/plugin/task/api/ShellCommandExecutor.java

```diff
@@ -112,6 +118,10 @@ protected void createCommandFileIfNotExists(String execCommand, String commandFi
                     }
                 }
             }
+
+            if (execCommand.startsWith("sqoop")) {
```

Comment:
May I ask whether we could do this with some other ways in `sqoop task plugin` instead of add a specific condition here for `sqoop`? ShellCommandExecutor is shared across different task plugins, it might not be a good practice and uneasy to maintain if we need to change it every time we need to mask something for a task plugin. The same for line 55-56.